components/footer/Footer.tsx

```diff
@@ -0,0 +1,92 @@
+import type { FC } from 'react';
+
+import Grid from '@mui/material/Grid';
+import Container from '@mui/material/Container';
+import Divider from '@mui/material/Divider';
+import Typography from '@mui/material/Typography';
+
+import type { Service } from '../../models';
+
+import Link, { type LinkProps } from '../common/link/Link';
+import RectangleIcon from '../common/icon-rectangle/RectangleIcon';
+import SiteLogo from '../common/navigation/SiteLogo';
+import { navigationRoutes } from '../common/navigation/NavigationLinks';
+
+import { FooterList, FooterListItem, IconRightContainer, PageContainer } from './styled-components';
+
+interface FooterProps {
+  services: Array<Service>;
```

Comment:
As far as I see, there are several sources of data:
- `services` - comes from a parent component passing via props
- `navigationRoutes` - using directly in this component
- `Hire Engineers` - plain text with code duplications

I would like to propose introducing a `data` prop with the following structure:
```ts
{
  services: [{service1}, {service2}, {serviceN}],
  navigations: [{navigation1}, {navigation2}, {navigationN}],
  hireEnfineers: [{hireEnfineer1}, {hireEnfineer2}, {hireEnfineerN}],
}
```
And then destructure `data` prop and use each property appropriately

Reply:
'+